### PR TITLE
Update latest embedded c release notes for May and June (no release)

### PR DESCRIPTION
**Pull request by ahsonkhan**
cc @czubair, @vhvb1989, @antkmsft

### CODE REVIEW

File: releases/2022-05/c.md [hunk 1]
```diff
@@ -8,38 +8,4 @@ repository: Azure/azure-sdk-for-c
...
-### _Package name_
-
-- Major changes only!
-
-## Latest Releases
-
-View all the latest versions of C packages [here][c-latest-releases].
-
-{% include refs.md %}
+No packaged released this month yet.
```
Comment (RickWinter) [inline releases/2022-05/c.md:11]:
```suggestion
No packages released this month yet.
```

File: releases/2022-06/c.md [hunk 2]
```diff
@@ -8,38 +8,4 @@ repository: Azure/azure-sdk-for-c
...
-### _Package name_
-
-- Major changes only!
-
-## Latest Releases
-
-View all the latest versions of C packages [here][c-latest-releases].
-
-{% include refs.md %}
+No packaged released this month yet.
```
Comment (RickWinter) [inline releases/2022-06/c.md:11]: Ditto from above

**Review by RickWinter — COMMENTED** [review]
Cala can you confirm the statement when we wont' release a package.

**Comment by ahsonkhan** [comment]
> Cala can you confirm the statement when we wont' release a package.

The verbatim is coming from here, and that needs to be updated:
https://github.com/Azure/azure-sdk/blob/main/_includes/releases/notes/common.md

**Review by czubair — APPROVED** [review]

File: releases/2022-05/c.md [hunk 3]
```diff
@@ -8,38 +8,4 @@ repository: Azure/azure-sdk-for-c
...
-### _Package name_
-
-- Major changes only!
-
-## Latest Releases
-
-View all the latest versions of C packages [here][c-latest-releases].
-
-{% include refs.md %}
+No packages released this month.
```
Comment (czubair) [inline releases/2022-05/c.md:11]:
```suggestion
No packages released yet for this month.
```

File: releases/2022-06/c.md [hunk 4]
```diff
@@ -8,38 +8,4 @@ repository: Azure/azure-sdk-for-c
...
-### _Package name_
-
-- Major changes only!
-
-## Latest Releases
-
-View all the latest versions of C packages [here][c-latest-releases].
-
-{% include refs.md %}
+No packages released this month.
```
Comment (czubair) [inline releases/2022-06/c.md:11]:
```suggestion
No packages released yet for this month.
```

**Comment by czubair** [comment]
> > Cala can you confirm the statement when we wont' release a package.
> 
> The verbatim is coming from here, and that needs to be updated (both the typo and the removal of "yet"): https://github.com/Azure/azure-sdk/blob/main/_includes/releases/notes/common.md

see this PR: https://github.com/Azure/azure-sdk/pull/4414

we want to keep the word "yet" to allow for OOB releases, which get rolled into the following month's release notes.

**Pull request merged**